_data/sitemap-apps.yml

```diff
@@ -25,20 +25,11 @@
     - title: App scopes
       link: page:app-scopes
 
-- key: apps-custom
-  title: Custom apps
-  entries:
-    - title: Use cases
-      link: page:apps-custom-use-cases
-
-    - title: Create an app
-      link: page:apps-create-custom
-
 - key: apps-develop
   title: Developing an app
   entries:
-    - title: Use cases
-      link: page:apps-use-cases
+    - title: Create a custom app
```

Comment:
What was the reason again that we say "Private apps" in the UI and "Custom apps" here? Isn't that confusing for a developer? (Maybe you can also explain that to me in person if it's easier ;) )